synergy-design-system/synergy-design-system
# chore: 🔧 Upgrade packages to latest

# Pull Request

## 📖 Description

This PR upgrades (most) packages to their latest stable version. Exceptions are:

- `semantic-release` (does not work with `semantic-release-monorepo@7.x`)
- `semantic-release-monorepo` (Patch does not apply for current 7.0.8 or 8.0.0)
- `change-case` (Multiple backwards-incompatible changes, needs deeper checks)

### 🎫 Issues

- Adresses #258 for the time being (except the packages mentioned above)

## 👩‍💻 Reviewer Notes

- As playwright was also upgraded, make sure to upgrade your local playwright browsers too (`pnpm exec playwright install`)

## 📑 Test Plan

- Check out the branch and issue an `pnpm i -r && pnpm lint && pnpm build`. Everything should work fine
- Start up all the demos and storybook and check if they behave correctly
- Validate there are no differences in wrapper project code when building the wrappers
- Check if storybook still has functioning knobs

## ✅ DoD
